ansible/roles/infra-ec2-create-inventory/default/main.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
This path should be `.../defaults/main.yml` (plural defaults)

Reply:
Is it this way upstream?